modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -83,18 +76,38 @@ def claim_meta
 
       def veteran_meta
         @evss_claim[:veteran] ||= {}
-        @evss_claim[:veteran][:currentVAEmployee] = @data.dig(:veteranIdentification, :currentVaEmployee)
+        # EVSS Docker needs currentlyVAEmployee, 526 schema uses currentVaEmployee
+        @evss_claim[:veteran][:currentlyVAEmployee] = @data.dig(:veteranIdentification, :currentVaEmployee)
         @evss_claim[:veteran][:emailAddress] = @data.dig(:veteranIdentification, :emailAddress, :email)
         @evss_claim[:veteran][:fileNumber] = @data.dig(:veteranIdentification, :vaFileNumber)
       end
 
       def map_date_to_obj(date)
```

Comment:
Should we write two tests for each of these date scenarios?

Reply:
Turns out I _assumed_ approximateDates were required by the Docker container but they are not.  So the whole dates checking for disabilities and secondary disabilities can be totally removed for this.